src/content/docs/apm/agents/java-agent/api-guides/guide-using-java-agent-api.mdx

```diff
@@ -37,7 +37,15 @@ For all available New Relic APIs, see [Intro to APIs](/docs/apis/getting-started
 
 ## Use the API [#api]
 
-To access the API class, add `newrelic-api.jar` to your application class path. The jar is in the New Relic Java agent's installation `zip` file. You can call the API when the Java agent is not running. The API methods are just stubs; the implementation is added when the Java agent loads the class.
+To access the API, download it from maven central via a build tool.
+
+For example with gradle:
+
+```groovy
+implementation 'com.newrelic.agent.java:newrelic-api:8.10.0'
+```
+
+You can call the API when the Java agent is not running. The API methods are just stubs; the implementation is added when the Java agent loads the class.
```

Comment:
```suggestion
You can call the API even without the agent running, but the methods won't work until the agent loads your application.
```

Does this revision work?

Reply:
Saying a method won't work is vague. I instead mentioned they are a no-op as it is clear they just won't do anything as supposed to say error out.